examples/generate-javascript-models/index.ts

```diff
@@ -10,7 +10,8 @@ const jsonSchemaDraft7 = {
       type: 'string',
       format: 'email'
     }
-  }
+  },
+  required: ['email'],
```

Comment:
To stay consistent I don't think we should add anything unnecessary to the inputs, as you already have a test in the JavaScriptGenerator 🙂